gradle-modules/src/main/kotlin/org.sonarsource.cloud-native.sonar-plugin.gradle.kts

```diff
@@ -27,12 +27,11 @@ val cleanupTask = tasks.register<Delete>("cleanupOldVersion") {
 
     // those variables need to be calculated in the configuration phase to enable Configuration cache
     // https://docs.gradle.org/8.12.1/userguide/configuration_cache.html#config_cache:requirements:disallowed_types
-    var projectName = "${project.name}-*.jar"
```

Comment:
I believe this should be just `val projectName = project.name`, because of the requirement for configuration cache

Reply:
LGTM :)